renderer/plugin.js

```diff
@@ -19,6 +19,12 @@ org.ekstep.contentrenderer.questionUnitPlugin = Plugin.extend({
     EkstepRendererAPI.addEventListener(this._manifest.id + ":hide", this.hideQuestion, this);
     EkstepRendererAPI.addEventListener(this._manifest.id + ":evaluate", this.evaluateQuestion, this);
     EkstepRendererAPI.addEventListener(this._manifest.id + ":rendermath", this.renderMath, this);
+    //Currently this plugin is regiesters twice upon rendering, Yet to find what's the issue, registering two event listerns with same function creating problems <Sivashanmugam kannan>
+    if(!EventBus.listeners["org.ekstep.questionunit:playaudio"]){
+      EkstepRendererAPI.addEventListener('org.ekstep.questionunit' + ":playaudio", this.handlePlayAudio, this);
```

Comment:
* Use constant object for all static string like pluginIds.
* Use events object for all the event names.

Reply:
constant: {
  PLUGIN_QUESTIONUNIT: "org.ekstep.questionunit"
  events: { 
   PLAY_AUDIO: "playaudio"
  }
}